plugin/build.gradle

```diff
@@ -29,6 +42,24 @@ ext {
     projectSubstitutions = [:]
     licenseFile = rootProject.file('LICENSE.txt')
     noticeFile = rootProject.file('NOTICE')
+
+    opensearchVersion = "${opensearch_version}"
+    isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+
+    runIntegTestWithSecurityPlugin= System.getProperty("security", "false") == "true" || System.getProperty("https", "false")   == "true"
+    if (runIntegTestWithSecurityPlugin) {
```

Comment:
Can't we add certificate as a backup if the download doesn't work?

Reply:
You can, but you will also be responsible to keep them up to date with security plugin.